File: DSA - Data Structures And Algorithm/Java/Creating a Graph and Displaying.java
import java.util.Scanner;

public class GraphInput {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter number of vertices: ");
        int n = sc.nextInt();

        int[][] graph = new int[n][n];

        for (int i = 0; i < n; i++) {
            for (int j = 0; j < n; j++) {
                System.out.print("Point " + (char)(65 + i) + " to " + (char)(65 + j) + " : ");
                graph[i][j] = sc.nextInt();
            }
        }

        System.out.println("Adjacency Matrix: ");
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < n; j++) {
                System.out.print(graph[i][j] + " ");
            }
            System.out.println();
        }
        
        sc.close();
    }
}

File: DSA - Data Structures And Algorithm/C/Creating a Graph and Displaying.c
// In this program we will create a graph and display it.

#include <stdio.h>

int main() {
    int n;
    printf("Enter number of vertices: ");
    scanf("%d", &n);

    int graph[n][n];

    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            printf("Point %c to %c: ", 65 + i, 65 + j); 
            scanf("%d", &graph[i][j]);
        }
    }

    printf("\nGraph Matrix:\n");
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            printf("%d ", graph[i][j]);
        }
        printf("\n");
    }

    return 0;
}
